wbooks/src/components/book-card/image/Image.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { styles } from './Image.styles';
+
+class Image extends Component {
+  render() {
+    if (this.props.image_url) {
+      return (
+        <img
```

Comment:
`<img className='${styles.image}' alt={'book' + this.props.bookId} src={this.props.image_url} />`
If doesnt have content (children) just close the tag.